sys/net/gnrc/transport_layer/tcp/gnrc_tcp_pkt.c

```diff
@@ -205,7 +205,10 @@ int _pkt_build(gnrc_tcp_tcb_t *tcb, gnrc_pktsnip_t **out_pkt, uint16_t *seq_con,
 
     /* Build network layer header */
 #ifdef MODULE_GNRC_IPV6
-    gnrc_pktsnip_t *ip6_snp = gnrc_ipv6_hdr_build(tcp_snp, NULL, (ipv6_addr_t *) tcb->peer_addr);
+    ipv6_addr_t *src_addr = (ipv6_addr_t *) tcb->local_addr;
```

Comment:
Just for clarifications: This is `NULL` or `::` if `tcb` is bound to "`ANY_ADDR`" right?